test/runtests.jl

```diff
@@ -78,13 +102,18 @@ Qf = Qf_rand * transpose(Qf_rand) + I
 
 mulbf    = build_QP_JuMP_model(Q,R,A,B, N;s0 = s0, sl = sl, su = su, ul = ul, uu = uu, Qf = Qf)
 dnlpulbf = LQDynamicData(s0, A, B, Q, R, N; sl = sl, su = su, ul = ul, uu = uu, Qf = Qf)
-lqpulbf  = LQDynamicModel(dnlpulbf)
+lqpulbfs  = LQDynamicModel(dnlpulbf;condense=false)
+lqpulbfc  = LQDynamicModel(dnlpulbf;condense=true)
 
 # Solve new problem with Qf matrix
 optimize!(mulbf)
-ipopt_ulbf  = ipopt(lqpulbf)
-madnlp_ulbf = madnlp(lqpulbf)
+ipopt_ulbfs  = ipopt(lqpulbfs)
+madnlp_ulbfs = madnlp(lqpulbfs)
+madnlp_ulbfc = madnlp(lqpulbfc)
```

Comment:
Rename as `solution_lq_condensed`? 

(in general explicit naming is better than implicit one)